src/host/config/KBoxConfigParser.cpp

```diff
@@ -54,6 +54,11 @@ void KBoxConfigParser::defaultConfig(KBoxConfig &config) {
 
   config.imuConfig.enabled = true;
   config.imuConfig.frequency = 20;
+  config.imuConfig.enabled = true;           // enable internal IMU sensor
+  config.imuConfig.enableHdg = true;         // true if values taken from internal sensor
+  config.imuConfig.enableHeelPitch = true;   // true if values taken from internal sensor
+  config.imuConfig.calHdg = 3;               // hdg valid, if calibration value greater equal
```

Comment:
I am still not convinced we need this. Besides you, and to a lesser extent me, nobody is going to understand what those values mean. 

If I agree to make 3 and 2 the defaults, are you ok removing this? ;)

Reply:
:-))
yes, this is ok.
As I can see the heading on the IMUPage now (independently from the SKUpdate, which means independently from any cal-value) it is ok for me.